static/cached_pair_addresses_docker.json

```diff
@@ -1,9 +1,9 @@
 [
+    "0x21b8065d10f73ee2e260e5b47d3344d3ced7596e",
     "0xb4e16d0168e52d35cacd2c6185b44281ec28c9dc",
-    "0xae461ca67b15dc8dc81ce7615e0320da1a9ab8d5",
     "0x0d4a11d5eeaac28ec3f61d100daf4d40471f1852",
+    "0x61b62c5d56ccd158a38367ef2f539668a06356ab",
+    "0xae461ca67b15dc8dc81ce7615e0320da1a9ab8d5",
     "0x3041cbd36888becc7bbcbc0045e3b1f144466f5f",
-    "0xd3d2e2692501a5c9ca623199d38826e513033a17",
-    "0xbb2b8038a1640196fbe3e38816f3e67cba72d940",
-    "0xa478c2975ab1ea89e8196811f51a7b7ade33eb11"
+    "0xe1573b9d29e2183b1af0e743dc2754979a40d237"
```

Comment:
I intentionally picked well known tokens for lite mode for demonstration, not based off volume/liquidity.

Reply:
Updated!